front/public/css/layout.css

```diff
@@ -218,12 +217,181 @@ h1 {
   width: 400px;
   height: 500px;
   border-radius: 15px;
-  background-color: lightgreen;
+  padding: 30px;
+  text-align: center;
+  background-color: #efefef;
 }
 .second-half {
   width: 500px;
   height: 500px;
-  background-color: lightblue;
+}
+.description-title {
+  font-size: 40px;
+  margin-bottom: 8px;
+}
+.description-image {
+  width: 270px;
+  height: 270px;
+  object-fit: cover;
+  border-radius: 15px;
+}
+.description-author {
+  text-transform: uppercase;
```

Comment:
Do you think we should do this in the JS code instead?

Reply:
we potentially could it is just easier to have all of the CSS files in one place but it definitely can be added to the JS code